.github/workflows/awesomebot.yml

```diff
@@ -0,0 +1,18 @@
+name: Check links in README.md
```

Comment:
We can have this in https://github.com/lima-vm/lima/blob/master/.github/workflows/test.yml